### ISSUE TITLE
Missing admin order view product data for grouped downloadable order items

### ISSUE THREAD

**Issue by wbyrnetx** [magento/magento2#4815]
Steps to reproduce
1. Install Magento 2.0.7.
2. Create a grouped product that has an associated downloadable product.
3. Place an order for that downloadable product.

Expected result
1. When viewing the order in the admin panel there will be no data under the "Product" column.
![screen shot 2016-06-01 at 7 40 11 am](https://cloud.githubusercontent.com/assets/4029452/15709790/3aac630c-27cc-11e6-9977-9d5d47c2a05f.png)

**Comment by wbyrnetx** [comment]
I put in a new pull request with a proposed solution (#4816). I just took the logic from 

Magento\Sales\Block\Adminhtml\Items\AbstractItems::getColumnHtml

And put it in in that name group renderer's _toHtml function. The main part that was replaced is

$renderer = $this->getRenderedBlock()->getColumnHtml($this->getItem(), $productType);

Which doesn't make much sense in the first place because getColumnHtml expects a column name as a second argument, not a productType.
